lib/notes.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const { default: HackMD } = require('@hackmd/api')
+const defaultNotesTemplate = require('./default-notes-template')
+
+module.exports.create = function createNote (notesTemplate, opts) {
+  const note = typeof notesTemplate === 'function' ? notesTemplate(opts) : notesTemplate || defaultNotesTemplate(opts)
+  const hackmd = new HackMD()
+
+  return hackmd.newNote(note)
+}
+
+async function getNotesTemplate (client, opts) {
+  const resp = await client.repos.getContents({
+    owner: opts.owner,
+    repo: opts.repo,
+    path: `.github/ISSUE_TEMPLATE/${opts.notesTemplate || 'meeting-notes.md'}`
```

Comment:
I think we should move this to `.github/` right?  Since it is technically not an issue template?

Reply:
I would say we may want to have our own folder in `.github/` - perhaps `.github/meet/`. But yes, I agree with this.